crates/timers/tests/web.rs

```diff
@@ -91,53 +91,57 @@ fn timeout_future_cancel() -> impl Future<Item = (), Error = wasm_bindgen::JsVal
 #[wasm_bindgen_test(async)]
 fn interval() -> impl Future<Item = (), Error = wasm_bindgen::JsValue> {
     let (mut sender, receiver) = mpsc::channel(1);
-    Interval::new(1, move || sender.try_send(()).unwrap()).forget();
+    Interval::new(1, move || {
+        if !sender.is_closed() {
+            sender.try_send(()).unwrap()
+        }
+    })
+    .forget();
```

Comment:
We shouldn't forget this -- that will cause the `setInterval` call to keep running even after the test is done.

Instead, we should do `let interval = Interval::new(...)` and then in the `and_then` closure, we should `drop(interval)` so that it gets cleared once the test is finished.

Reply:
Ah yes, good call. Done!